plugins/env/app/views/samson_env/_fields.html.erb

```diff
@@ -5,9 +5,11 @@
   </legend>
 
   <% if ENV['DEPLOYMENT_ENV_REPO'] %>
-    <%= form.input :use_env_repo, label: "Use GitHub repository #{ENV['DEPLOYMENT_ENV_REPO']} to manage deployment environment variables",
-                   as: :check_box
-    %>
+    <% project = form.object %>
```

Comment:
can use `@project`

Reply:
Done but should I also have changed this for the other code blocks?